app/assets/stylesheets/question.scss

```diff
@@ -9,10 +9,8 @@
     border-color: $govuk-error-colour;
     background-color: $govuk-error-colour;
   }
-
-  &--error, &--success {
-    p {
-      margin: 0;
-    }
+  
+  p {
```

Comment:
We do need the `--error` otherwise @gharteyj1 will complain of additional spacing in errors

Reply:
Check out my screenshots - they all seem fine to me?